metron-platform/metron-indexing/metron-indexing-common/src/test/java/org/apache/metron/indexing/dao/metaalert/MetaAlertIntegrationTest.java

```diff
@@ -246,18 +223,18 @@ public void shouldSortByThreatTriageScore() throws Exception {
     srAsc.setSort(Collections.singletonList(sfAsc));
     result = metaDao.search(srAsc);
     results = result.getResults();
-    Assert.assertEquals("message_1", results.get((0)).getSource().get(Constants.GUID));
-    Assert.assertEquals("meta_active_0", results.get((1)).getSource().get(Constants.GUID));
-    Assert.assertEquals(2, results.size());
+    assertEquals("message_1", results.get((0)).getSource().get(Constants.GUID));
+    assertEquals("meta_active_0", results.get((1)).getSource().get(Constants.GUID));
+    assertEquals(2, results.size());
   }
 
   @Test
   public void getAllMetaAlertsForAlertShouldThrowExceptionForEmptyGuid() throws Exception {
     try {
       metaDao.getAllMetaAlertsForAlert("");
-      Assert.fail("An exception should be thrown for empty guid");
+      fail("An exception should be thrown for empty guid");
```

Comment:
Might be worth using the new idiom for exception testing here.

Reply:
Cleaned these up throughout. There's a few `fail` left, but they shouldn't be things that are just primarily checking an exception.